www/simple.php

```diff
@@ -253,9 +253,13 @@ function getScheme() {
 	$hubic_user = $_GET['hubic_user'];
 	$hubic_password = $_GET['hubic_password'];
 
+        if (is_array($clients[$client]) && array_key_exists('autoredirect_uri', $clients[$client])) {
```

Comment:
Thanks for the note. I've been a long time ago since I used php seriously. And before I think  gave error or a warning if you try to check from a nonexistent array, that's why I check the array first. But now I test both ways and works.